docs/guide/quickstart.rst

```diff
@@ -21,8 +21,9 @@ The view requires :code:`HtmxViewMixin`
 .. code-block:: python
 
     from hx_requests.views import HtmxViewMixin
+    from django.views.generic.base import View
 
-    class MyView(HtmxViewMixin):
+    class MyView(View, HtmxViewMixin):
```

Comment:
I think the Mixin needs to come prior to the view. Though I am not positive about that because it looks like `View` does not have Get and Post methods. But for consistency where in most cases it will need to come first I think the docs should put The mixin before View

Reply:
You are correct.

Reply:
This does bring up an issue where `get` or `post` are not defined in a view because like you said (correctly) the `View` class does not implement any of the HTTP methods so calling super().get or post will raise an exception. This is really a separate issue. I'm opening a separate issue here #45 to address this.